programs/cleos/main.cpp

```diff
@@ -145,8 +145,8 @@ bool   tx_dont_broadcast = false;
 bool   tx_skip_sign = false;
 bool   tx_print_json = false;
 
-uint32_t tx_cf_cpu_usage = 0;
-uint32_t tx_net_usage = 0;
+uint32_t max_cpu_usage = 0;
```

Comment:
while the `tx_` prefix isn't pretty it is conventional with the parameters above like `tx_skip_sign` and `tx_print_json` we should maintain the convention or change it universally.